sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -1271,7 +1271,7 @@ case class Pow(left: Expression, right: Expression)
        4
   """,
   since = "1.5.0",
-  group = "math_funcs")
+  group = "bitwise_funcs")
```

Comment:
Actually we'll need to update Scaladoc, and Python docs too. functions.scala and funcions.py and functions.R

Reply:
will do for python functions, in a separate PR